Documentation/Contributors/CodingGuide/README.md

```diff
@@ -135,6 +135,25 @@ function Model(options) {
 
 * Text files, including JavaScript files, end with a newline to minimize the noise in diffs.
 
+## Code Linting
+
+We use [ESLint](http://eslint.org/) to check that our code adheres to syntax and style guidelines. In order to keep our ESLint guidelines
+the same across repositories, we use a [shareable config](http://eslint.org/docs/developer-guide/shareable-configs):
+[`eslint-config-cesium`](https://www.npmjs.com/package/eslint-config-cesium).
+
+The two common configurations defined in `eslint-config-cesium` are `browser` (used for Javascript and HTML which will run in the browser) and `node` (used
```

Comment:
I'm lost here.  Should this link to something we publish to npm?  To https://github.com/AnalyticalGraphicsInc/cesium/blob/master/.eslintrc.json / https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Tools/eslint-config-cesium/.eslintrc.json?

I'm also not sure if this belongs in this guide.  The updates I asked for to this guide were specific code guidelines and links to the specific ESLint pages.

ESLint configuration may be better left elsewhere or in an appendix.